### PR TITLE
[15.0][MOV] rename mrp_production_cancel_manager to mrp_production_buttons_manager

### PR DESCRIPTION
**Pull request by IriaAlonso**
- Rename mrp_production_cancel_manager to mrp_production_buttons_manager
- Generalize deny access to buttons to Non-Managers like "scrap", "unreserve", "check availability" and "mark as done"

@dalonsod

### CODE REVIEW

File: mrp_production_buttons_manager/views/mrp_production_view.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_cancel']" position="attributes">
+                <attribute name="groups">mrp.group_mrp_manager</attribute>
+            </xpath>
+            <xpath expr="//button[@name='action_cancel'][2]" position="attributes">
+                <attribute name="groups">mrp.group_mrp_manager</attribute>
+            </xpath>
+            <xpath expr="//button[@name='button_mark_done'][2]" position="attributes">
```
Comment (dalonsod) [inline mrp_production_buttons_manager/views/mrp_production_view.xml:18]: This is weird... why start for `button_mark_done` second appearance in view?

**Comment by IriaAlonso** [comment]
@dalonsod done

**Comment by dalonsod** [comment]
Git history for the most files will be properly reserved, but not for these two ones:

imagen

Could you check?

**Review by dalonsod — APPROVED** [review]
Well keep this operations splitted in two commits, :+1: 

I've only improved second commit title and detailed description, in order to make it more descriptive, please take a look @IriaAlonso 

Merging!!!

**Pull request merged**